modules/charts/src/components/Provider/Provider.tsx

```diff
@@ -0,0 +1,119 @@
+import { useArrangerData } from '@overture-stack/arranger-components';
+import React, { createContext, ReactElement, useContext, useMemo, useRef } from 'react';
+
+import { useNetworkQuery } from '#hooks/useNetworkQuery';
+import { createChartColors } from './Colors/colors';
+
+type ChartContextType = {
+	theme: {};
+	registerChart: ({ fieldName }: { fieldName: string }) => void;
+	deregisterChart: ({ fieldName }: { fieldName: string }) => void;
+	getChartData: ({ fieldName }: { fieldName: string }) => {
+		isLoading: boolean;
+		isError: boolean;
+		// TODO: Map<string, Aggregtions | NumericAggregations>
+		data: Map<string, {}> | undefined;
+	};
+};
+
+export const ChartsContext = createContext<ChartContextType | null>(null);
+
+type ChartsProviderProps = React.PropsWithChildren<{
```

Comment:
seems like `PropsWithChildren` is the only bit that relies on the larger React import (which also includes all sorts of other properties, likely including the ones destructured at the same time).

Is there a particular reason why this one is not just destructured like the others?